src/features/api-keys/lib.ts

```diff
@@ -158,17 +178,17 @@ export type PartialCreateKey = (
 export const createProvisioningApiKey: PartialCreateKey = _.partial(
 	createApiKey,
 	'application',
-	'provisioning-api-key',
+	KeyMetadata.PROVISIONING.role,
 );
 export const createDeviceApiKey: PartialCreateKey = _.partial(
 	createApiKey,
 	'device',
-	'device-api-key',
+	KeyMetadata.DEVICE.role,
 );
 export const createNamedUserApiKey: PartialCreateKey = _.partial(
 	createApiKey,
 	'user',
-	'named-user-api-key',
+	KeyMetadata.USER.role,
```

Comment:
Let's revert all changes in this file since they are not required.
PS: An enum would have been enough for this.